metagenomics/ont/samplesheet_test.csv

```diff
@@ -0,0 +1,3 @@
+sample,fastq_1,fastq_2
+oma,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/ont/oma_subset_reads.fq.gz,
+eligo,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/ont/eligo_subset_reads.fq.gz,
```

Comment:
is it ok to have the cheese names publicly represented in the sample name and fq file name?

Reply:
I should ask Emily, easy to change if not

Reply:
updated